shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -53,4 +53,8 @@ FLUTTER_EXPORT
     NS_DESIGNATED_INITIALIZER;
 - (nonnull instancetype)initWithCoder:(nonnull NSCoder*)nibNameOrNil NS_DESIGNATED_INITIALIZER;
 
+- (void)handleMethodCall:(nonnull FlutterMethodCall*)call result:(nonnull FlutterResult)result;
```

Comment:
Why are you making this public?

Reply:
Good catch, I thought this was necessary to call it from the test, but it seems to work without it. I'll remove it.